src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.star {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(1),
```

Comment:
You can shorten this code a lot if you use nth-child(-n + 1) etc.  https://css-tricks.com/how-nth-child-works/